test/users/npadmana/mpi/c2chapel.py

```diff
@@ -0,0 +1,83 @@
+""" A C to Chapel parser to help automate writing bindings.
+
+This is built on top of pycparser :
+  https://github.com/eliben/pycparser
+(pycparser is also part of the Anaconda distribution)
+
+Some parts of this code may need to be special cased for particular bindings,
+and the resulting bindings may require some human intervention.
+"""
```

Comment:
It would be nice to have instructions, here or in the README file, for how the extern declarations can be generated again (e.g. download HTMLs, run this script, etc).